modules/virt-creating-a-service-backed-by-virtual-machines.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-creating-a-service-backed-by-virtual-machines_{context}"]
+
+= Creating a service backed by virtual machines
+
+To create a service backed by a virtual machine, you start the `VirtualMachineInstance`,
+if it is not already running. Once the instance is running, you connect to the instance, and
```

Comment:
How? Do you mean ssh to the VM? What are the `instance`s reffering here?

Reply:
Changed this based on @maiqueb suggestions:

You can create and expose a service from a running virtual machine.

To create a service backed by a virtual machine, start the `VirtualMachine`,
if it is not already running. Once the virtual machine is running, create a service
object to expose the service.

Reply:
Not sure this only adds more confusion, but you can create a service on a VM *before* starting it - it will obviously only be reachable once the VM is running.

Your call.

Reply:
Yeah, I think presenting one way is probably best.